pkg/provision/workspace/ssh-askpass.sh

```
@@ -1,4 +1,11 @@
 #!/bin/sh
+# Only handle passphrase prompts - exit silently for other prompts (e.g., HTTPS git repo username/password request)
+case "$1" in
```

Shellcheck (0.11.0)

[info] The mentioned syntax error was in this case expression.

(SC1009)